sources/subs/Profile.subs.php

```diff
@@ -114,7 +114,7 @@ function setupProfileContext($fields, $hook = '')
 			}
 
 			// Was there an error with this field on posting?
-			if (isset($context['profile_errors'][$field]))
+			if (isset($context['post_errors '][$field]))
```

Comment:
There is a space at the end of "post_errors", before the closing single quote. ;)